qt/applications/workbench/workbench/plotting/figuremanager.py

```diff
@@ -82,10 +82,10 @@ def deleteHandle(self, _, workspace):
         all_axes = self.canvas.figure.axes
         if not all_axes:
             return
-        empty_axes = True
+        empty_axes = False
         for ax in all_axes:
             if isinstance(ax, MantidAxes):
-                empty_axes = empty_axes & ax.remove_workspace_artists(workspace)
+                empty_axes = ax.remove_workspace_artists(workspace)
```

Comment:
Is this now missing the case where there is a subplot with multiple `Axes` instances and only 1 of the axes becomes empty? The window would now close wouldn't it?